scripts/fetch-release-changelog.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as fs from 'fs';
+import * as process from 'process';
+import * as https from 'https';
+
+/**
+ * Take the `pullRequestNumber` output of the `changesets/action` GitHub action
+ * on standard input and return the CHANGELOG from the PR description in stdout.
+ *
+ * https://github.com/changesets/action#outputs
+ */
+
+const pullRequestNumber = fs.readFileSync(process.stdin.fd, 'utf-8').trim();
+if (!pullRequestNumber || isNaN(parseInt(pullRequestNumber, 10))) {
+  console.error(`No pullRequestNumber from changeset action.`);
+  process.exit(1);
+}
+const GITHUB_GET_PR = `https://api.github.com/repos/lit/lit/pulls/${pullRequestNumber}`;
+
+const response = await new Promise((resolve) => {
+  let finalData = '';
+  https
+    .get(
+      GITHUB_GET_PR,
+      {
+        headers: {
+          accept: 'application/vnd.github+json',
+          'User-Agent': 'Lit',
+        },
+      },
+      (res) => {
+        res.on('data', (d) => {
+          finalData += d;
+        });
+
+        res.on('end', () => {
+          resolve(JSON.parse(finalData));
+        });
+      }
+    )
+    .on('error', (e) => {
+      console.error(e);
+      process.exit(1);
+    });
+});
+
+const {body} = response;
+
+// Strip the preamble from the PR description, and only capture the changelog
+// contents.
+const headingIdx = body.indexOf('# Releases');
+console.log(body.slice(headingIdx === -1 ? 0 : headingIdx));
```

Comment:
TIL about [process substitution](http://www.gnu.org/software/bash/manual/html_node/Process-Substitution.html#Process-Substitution)

Is there a benefit to doing this and providing to `release-image` with it as opposed to just writing to a fixed filename and providing that filename to `release-image`?

Reply:
Discussed offline. The main benefit is that process substitution handles the temporary intermediate file without needing to maintain that reference in a temporary variable. Because the scope of the script is small, this is probably fine.

If the scope increased and it could become necessary to add additional console logs, then this decision should be revisited.